environment.js

```diff
@@ -2,25 +2,37 @@ var path = require('path');
 var fs = require('fs');
 var mkdirp = require('mkdirp');
 var glob = require('glob');
+var q = require('q');
 
-export.ensureDirectory = function(directoryName) {
+exports.ensureDirectory = function(directoryName) {
   if (!fs.existsSync(directoryName)) {
     mkdirp.sync(directoryName);
   }
 };
 
-export.loadSpecifications = function(mocha, directory) {
-   function addSpecs(error, files) {
+
+exports.loadSpecifications= function(directory, mocha) {
+  var deferred = q.defer();
```

Comment:
I thought it might be easier to read like this: https://gist.github.com/cutandpastey/fa662001ffb4b233226a but im too used to es6 now so this looks pretty messy as well.